### PR TITLE
feat: add aptos bot tests

### PR DESCRIPTION
**Pull request by jccguimaraes**
### ✅ Checklist

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Adds Speculos Bot tests for the Aptos blockchain

### ❓ Context

- **JIRA or GitHub link**: [https://ledgerhq.atlassian.net/browse/LIVE-14004](https://ledgerhq.atlassian.net/browse/LIVE-14004)

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: libs/coin-modules/coin-aptos/src/test/cli.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    name: "fee",
+    type: String,
+    desc: "how much fee",
+  },
+  {
+    name: "tag",
+    type: Number,
+    desc: "aptos tag",
+  },
+];
```
Comment (semeano) [inline libs/coin-modules/coin-aptos/src/test/cli.ts:17]: Honest question: how do we know which options we need to add here?

Comment (joaoccmartins) [reply libs/coin-modules/coin-aptos/src/test/cli.ts:17]: Honest answer: no clue 😄 
I essentially just copied from another coin and changed where it made sense. We need to have this validated by @hedi-edelbloute whenever he's back (which I'd say, doesn't block the merge).

Comment (qperrot) [reply libs/coin-modules/coin-aptos/src/test/cli.ts:17]: I think you should keep this logic inside `ledger-live-common/src/families/aptos` Then we will move all the logic inside the coin-module (during the coin modularization)

**Review by hedi-edelbloute — APPROVED** [review]
Bot issue looks not related to the pull request itself, maybe on nano app side

**Pull request merged**